src/hotspot/os/linux/os_linux.cpp

```diff
@@ -4480,19 +4488,19 @@ void os::Linux::numa_init() {
   // bitmask when externally configured to run on all or fewer nodes.
 
   if (!Linux::libnuma_init()) {
-    FLAG_SET_ERGO(UseNUMA, false);
-    FLAG_SET_ERGO(UseNUMAInterleaving, false); // Also depends on libnuma.
+    disable_numa("NUMA support is disabled as libnuma failed to initialize");
```

Comment:
The `disable_numa()` call should only include the reason:
```suggestion
    disable_numa("Failed to initialize libnuma");
```